Add devnet connection instructions (#80)
filecoin-project/filecoin-docs: docs/how-to/build-interacting-with-the-network.md

- [Lotus Docker Image](https://github.com/openworklabs/filecoin-docker) for a simple Lotus node Docker container
- [Filecoin Chart](https://github.com/openworklabs/filecoin-chart) for more complex Lotus node architectures managed with Kubernetes

> ## Developer network

Reviewer: ```suggestion
## Devnet
```

> In order to speed up seal times and make testing easier, a live developer network is now available for use. It features 512MB sectors, which seal much faster than in the full Testnet version.

Reviewer: ```suggestion
In order to make testing easier, a live devnet (developer network) is now available for use. It features 512MB sectors and reduced proofs parameters, so sealing is much faster than in the full Testnet version.
```

> In order to set up the developer network:

Reviewer: ```suggestion
To connect to the Devnet:
```
